Frends.SFTP.DownloadFiles/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## [2.5.2] - 2023-01-31
+### Added
+- Added cancellation token to the method listing the source files. 
+- Modified method's name from GetSourceFiles to ListSourceFiles.
```

Comment:
This is internal implementation detail, which is not really important from user-facing changeling perspective, so it should be omitted.